build.gradle.kts

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id("com.hivemq.extension")
+    id("com.github.hierynomus.license")
+    id("org.asciidoctor.jvm.convert")
+}
+
+group = "com.hivemq.extensions"
+description = "HiveMQ extension to utilize Prometheus for monitoring"
+
+hivemqExtension {
+    name = "Prometheus Monitoring Extension"
+    author = "HiveMQ"
+    priority = 1000
+    startPriority = 0
```

Comment:
@DC2-DanielKrueger the default start priority is 1000, so we should not change it